src/main/java/shortestpath/ShortestPathPlugin.java

```diff
@@ -145,23 +145,6 @@ protected void shutDown() {
         overlayManager.remove(pathMapTooltipOverlay);
     }
 
-    @Subscribe
-    public void onConfigChanged(ConfigChanged event) {
```

Comment:
Since you removed this event you should also remove the related import in line 37:
`import net.runelite.client.events.ConfigChanged;`